### ISSUE TITLE
Getting "reporter [0] must have an init method" with Hapi 11.1

### ISSUE THREAD

**Issue by gajjargaurav** [atroo/hapi-good-logstah#1]
I am using hapi-good-logstash with Hapi 11 and I get follwoing error,

``` js
Error: reporter [0] must have an init method
    at Object.exports.assert (myapplication/node_modules/hoek/lib/index.js:731:11)
    at myapplication/node_modules/good/lib/monitor.js:162:14
    at iterate (myapplication/node_modules/items/lib/index.js:35:13)
    at Object.exports.serial (myapplication/node_modules/items/lib/index.js:38:9)
    at internals.Monitor.start (myapplication/node_modules/good/lib/monitor.js:126:11)
    at Object.exports.register (myapplication/node_modules/good/lib/index.js:17:20)
    at Object.target [as register] (myapplication/node_modules/hapi/node_modules/joi/lib/object.js:80:30)
    at each (myapplication/node_modules/hapi/lib/plugin.js:315:14)
    at iterate (myapplication/node_modules/hapi/node_modules/items/lib/index.js:36:13)
    at done (myapplication/node_modules/hapi/node_modules/items/lib/index.js:28:25)
    at Object.exports.register (myapplication/node_modules/lout/lib/index.js:108:5)
    at Object.target [as register] (myapplication/node_modules/hapi/node_modules/joi/lib/object.js:80:30)
    at each (myapplication/node_modules/hapi/lib/plugin.js:315:14)
    at iterate (myapplication/node_modules/hapi/node_modules/items/lib/index.js:36:13)
    at done (myapplication/node_modules/hapi/node_modules/items/lib/index.js:28:25)
    at Object.exports.register (myapplication/node_modules/inert/lib/index.js:33:12)
```

I think it is down to the fact that hapi-good-logstash uses [GoodReporter](https://github.com/hapijs/good-reporter)

**Comment by phal0r** [comment]
hi @gajjargaurav,

thanks for reporting this. Right now we are using Hapi 8.x which uses the old good version with support for the GoodReporter interface. Since we are focussing on being up-to-date with our module dependencies, we will definitely adapt to the new standard, but it may take some time until we finish the next iteration. So any help on updating to the next major release would be appreciated.

**Comment by phal0r** [comment]
hi @gajjargaurav 

we have released a new version (1.0.0) which supports good 6 and newer. This should fix your issue.
